OracleCoherence/Dockerfile

```diff
@@ -0,0 +1,90 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Coherence 12.1.3 Standalone Distribution
+# 
+# REQUIRED BASE IMAGE TO BUILD THIS IMAGE
+# ---------------------------------------
+# Make sure you have oraclelinux:7.0 Docker image installed.
+# Visit for more info: 
+#  - http://public-yum.oracle.com/docker-images/
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_12.1.3.0.0_coherence_Disk1_1of1.zip 
+#     Download the Standalone installer from http://www.oracle.com/technetwork/middleware/coherence/downloads/index.html
+#
+# (2) jdk-8u25-linux-x64.rpm
+#     Download from http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ sudo sh build.sh
+#
+
+# Pull base image
+# ---------------
+FROM oraclelinux:7.0
```

Comment:
This should be: ``FROM oraclelinux:7`` so that it always pulls the latest version of Oracle Linux 7 and doesn't get stuck on 7.0 when 7.1 is released.